components/radio/stories/template.js

```diff
@@ -40,6 +43,10 @@ export const Template = ({
 				readOnly=${isReadOnly ? "readonly" : ""}
 				?checked=${isChecked}
 				?disabled=${isDisabled}
+				@change=${() => {
+					if (isDisabled) return;
+					updateArgs({ isChecked: !isChecked });
+				}}
```

Comment:
This `@change` was being used on Checkbox as well. In order to add a control to switch between checkbox groups and radio groups without a hooks error ("rendered more/fewer hooks than previous render"), Checkbox and Radio need to use the same number of hooks, so `useArgs()` was added here.